AbstractModels/AbstractUser.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * Model object generated by: Skipper (http://www.skipper18.com)
+ * Do not modify this file manually.
+ */
+
+namespace App\Models\AbstractModels;
+
+use Illuminate\Database\Eloquent\Model;
+
+abstract class AbstractUser extends Model
```

Comment:
Most important, this class is now an abstract class and the class name reflects that

Reply:
done